x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/service/RestGetServiceAccountTokensAction.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.security.rest.action.service;
+
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.license.XPackLicenseState;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.RestToXContentListener;
+import org.elasticsearch.xpack.core.security.action.service.GetServiceAccountTokensAction;
+import org.elasticsearch.xpack.core.security.action.service.GetServiceAccountTokensRequest;
+import org.elasticsearch.xpack.security.rest.action.SecurityBaseRestHandler;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.elasticsearch.rest.RestRequest.Method.GET;
+
+public class RestGetServiceAccountTokensAction extends SecurityBaseRestHandler {
+
+    public RestGetServiceAccountTokensAction(Settings settings, XPackLicenseState licenseState) {
+        super(settings, licenseState);
+    }
+
+    @Override
+    public List<Route> routes() {
+        return List.of(
+            new Route(GET, "/_security/service/{namespace}/{service}/credential")
```

Comment:
I know you previously mentioned that you preferred `/service_account/` and `{service-name}`.
Do you still? Should we discuss?

Reply:
I still slightly prefer `/service_account/` over `/service/` (`{service}` is fine). The main reason is that I don't know whether in the future we would need a more accurate usage of `/service/`, e.g. configures some service within the security domain, say `PUT /_security/service/builtin_directory/config`. But it may not be an issue at all because it can simply just be `PUT /_security/builtin_directory/config`. So I think it is a minor personal preference and I am happy to stick to the current design.